packages/plugin-api-docgen/package.json

```diff
@@ -45,7 +45,7 @@
   "peerDependencies": {
     "react": ">=17.0.0",
     "react-router-dom": "^6.8.1",
-    "@rspress/core": "^1.0.2",
+    "@rspress/core": "workspace:*",
```

Comment:
Using `workspace:*` will generate a fixed version number, which forces the plugin to use the same version as `@rspress/core`.

It is usually more appropriate to define a version range in peerDependencies, such as `workspace:^1.26.3` (Modern.js uses this format), and changeset will bump the version in `workspace:^1.26.3`.

Reply:
Isn't a fixed version better? Since we don't have a CI process to guarantee that the plugin can work with another version of `@rspress/core`.

Reply:
Fixed version is too strict, in most cases Rspress plugins can work with future versions of `@rspress/core`. As we do not remove any plugin API in non-major releases.

Most plugin systems in the community choose a loose peer dependency declaration, usually declaring one or more major versions.

Examples:

- Vite: https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/package.json#L47-L49
- webpack plugin: https://github.com/jantimon/html-webpack-plugin/blob/main/package.json#L72-L75
- webpack loader: https://github.com/webpack-contrib/sass-loader/blob/master/package.json#L44-L50
- babel: https://github.com/babel/babel/blob/main/packages/babel-plugin-syntax-typescript/package.json#L23-L25

Reply:
Changeset will bump a major version for plugin after we add a minor or major changset for core if we use a fixed version in a peerdeps.

Reply:
OK I'll use `workspace:^`.

Reply:
updated